emloop/cli/args.py

```diff
@@ -24,6 +24,7 @@ def get_emloop_arg_parser(add_common_arguments: bool=False) -> ArgumentParser:
     train_parser = subparsers.add_parser('train', description='Start emloop training from the ``config_file``.')
     train_parser.set_defaults(subcommand='train')
     train_parser.add_argument('config_file', help='path to the config file')
+    train_parser.add_argument('--rm', action='store_true', help='Delete directory after training finished')
```

Comment:
```suggestion
    train_parser.add_argument('--rm', action='store_true', help='Delete directory after training is finished')
```

Reply:
```suggestion
    train_parser.add_argument('--rm', action='store_true', help='Delete directory after training finishes')
```

Reply:
I am not 100% sure about this but *after training finished* feels weird to me (please extend this to the other docstrings)